### PR TITLE
Replication Transforms - add rename and mask directives

### PR DESCRIPTION
**Pull request by seanfeld**
Adds `mask` and `rename` transforms to the submenu for replication transformations

### CODE REVIEW

File: app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/addToTransforms.ts [hunk 1]
```diff
@@ -26,3 +25,8 @@ export const addRenameToTransforms = (transform: IColumnTransformation): string
   const { columnName, directive } = transform;
   return `rename ${columnName} ${directive}`;
 };
+
+export const addMaskToTransforms = (transform: IColumnTransformation): string => {
+  const { columnName, directive } = transform;
+  return `mask ${columnName} ${directive}`;
```
Comment (njbriggs) [inline app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/addToTransforms.ts:31]: Should column names be preceded by `:`?

Comment (seanfeld) [reply app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/addToTransforms.ts:31]: @njbriggs afaik this is how wrangler does them `directiveName columnName directiveText` just in a string - it works for rename that way tested all the way through.

Comment (njbriggs) [reply app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/addToTransforms.ts:31]: We'd like to consistently use `:` for column names in Wrangler: https://cdap.atlassian.net/browse/CDAP-17765

Comment (seanfeld) [reply app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/addToTransforms.ts:31]: I'd have to look at the code for the plugin and do some testing to validate that works. According to this https://docs.google.com/document/d/17yJjaLs2czElogB9RizTqPIGPB8kcIU9vNjwXfVza3I/edit?resourcekey=0-wdGszG0eqOtnLyBPSHwQhA#bookmark=id.yrlg98x0gyh2 its supposed to be without colons.

Comment (seanfeld) [reply app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/addToTransforms.ts:31]: @njbriggs can we get this code in and then i can do another pr addressing that if my testing etc works. It works without the `:` for now

Comment (njbriggs) [reply app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/addToTransforms.ts:31]: OK then, let's get this merged.

**Review by njbriggs — APPROVED** [review]

File: app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/addToTransforms.ts (hunk 1 above)
Comment (seanfeld) [reply app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/addToTransforms.ts:31]: cool - made this jira task to keep track of it https://cdap.atlassian.net/browse/CDAP-18698

**Pull request merged**